releases/dev/app.yml

```diff
@@ -21,13 +21,13 @@ spec:
       replicaCount: 2
       image:
         repository: pinscommonukscontainersdefault.azurecr.io/appeals-service-api
-        tag: 1.0.0
+        tag: 1.2.0
 
     formsWebApp:
       replicaCount: 2
       image:
         repository: pinscommonukscontainersdefault.azurecr.io/forms-web-app
-        tag: 1.2.0
+        tag: 1.3.0
```

Comment:
mate- these are just magic numbers changing...
;)

Reply:
They are that. This will hopefully be the last manual one before I get the automated released fixed (my next task)